docs/general-usage/interfaces.rst

```diff
@@ -0,0 +1,166 @@
+    nextSiblings(limit: PositiveInt, offset: PositiveInt, order: String, searchQuery: String, id: ID): [PageInterface]
+    previousSiblings(limit: PositiveInt, offset: PositiveInt, order: String, searchQuery: String, id: ID): [PageInterface]
+    descendants(limit: PositiveInt, offset: PositiveInt, order: String, searchQuery: String, id: ID): [PageInterface]
+    ancestors(limit: PositiveInt, offset: PositiveInt, order: String, searchQuery: String, id: ID): [PageInterface]
+
+
+Any custom ``graphql_fields`` added to your specific Page models will be available here via the 'on' spread operator and
+the name of the model:
+
+::
+
+    query {
+        pages {
+            ...on BlogPage {
+                the_custom_field
+            }
+        }
+    }
+
+You can change the default ``PageInterface`` to your own interface by changing the
+:ref:`PAGE_INTERFACE<page interface settings>` setting.
+
+As mentioned above there is both a plural ``pages`` and singular ``page``
+field on the root Query type that returns a ``PageInterface``.
+
+The plural ``pages`` field (as do all plural fields)
+accepts the following arguments:
+
+::
+
+    id: ID
+    limit: PositiveInt
+    offset: PositiveInt
+    order: String
+    searchQuery: String
+    contentType: String           #  comma separated list of content types in app.Model notation
+    inSite: Boolean
+    ancestor: PositiveInt         # ID of ancestor page to restrict results to
+    parent: PositiveInt           # ID of parent page to restrict results to
+
+
+The singular ``page`` field accepts the following arguments:
+
+::
+
+    id: ID                        # Can be used on it's own
+    slug: String                  # Can be used on it's own
+    urlPath: String               # Can be used on it's own
+    token: String                 # Must be used with one of the others. Usually contentType
+    contentType: String           # Can be used on it's own
+    inSite: Boolean               # Can be used on it's own
+
+
+
+``StreamFieldInterface``
+------------------------
+
+``StreamFieldInterface`` is the default interface for all Wagtail StreamField block models. It exposes the following
+fields, following the base fields and properties available to the Wagtail StreamField blocks:
+
+::
+
+    id: ID
+    blockType: String!
+    field: String!
+    rawValue: String!
+
+Note that blocks subclassing `StreamBlock <https://docs.wagtail.org/en/stable/topics/streamfield.html#streamblock>`_
+and `StructBlock <https://docs.wagtail.org/en/stable/topics/streamfield.html#structblock>`_ have an additional property
+in the interface:
+
+::
+
+    blocks: [StreamFieldInterface!]!  # a list of blocks in the StreamBlock or StructBlock
+
+
+
+Adding your own interfaces
+--------------------------
+
+To add your own interfaces to your model, define the ``graphql_interfaces`` attribute on it. The attribute should be
+a tuple of interfaces to add to the model.
```

Comment:
**Nitpick:** Would this look better as a list? Django historically used tuples for class attributes on models, forms, ModelAdmins, and such, but have been transitioning to lists. [Python docs](https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences) also have this to say:

> Tuples are immutable, and usually contain a heterogeneous sequence of elements that are accessed via unpacking (see later in this section) or indexing (or even by attribute in the case of namedtuples). Lists are mutable, and their elements are usually homogeneous and are accessed by iterating over the list.

Reply:
Both will work as we're unpacking with `*`. Can tweak the language to say tuple/list

Reply:
tweaked to suggest list first, tuple second